### ISSUE TITLE
Domains and Subdomains

### ISSUE THREAD

**Issue by marcussaad** [juanpablob/angular-environment#3]
Hi @juanpablob ,

I'm playing with the following config:

``` envServiceProvider.config({
          domains: {
              local: 'localhost',
              dev: ['dev.mydomain.com'],
              qa: ['qa.mydomain.com'],
              production: ['mydomain.com']
          },
          vars: {
              local: {
                  api_url: 'http://127.0.0.1:8000/',
              },
              dev: {
                  api_url: 'http://dev.mydomain.com:8001/',
              },
              qa:{
                  api_url: 'http://qa.mydomain.com:8001/'
              },
              production: {
                  api_url: 'http://mydomain.com:8001/',
              }
          }
      });
     envServiceProvider.check();
```

When visiting qa.mydomain.com, it still queries the url for my production env, and the same for dev.

**Comment by juanpablob** [comment]
@marcussaad first of all thank you for the information, but could you please describe again the problem? I didn't get very well the issue. Thank you!

**Comment by marcussaad** [comment]
My pleasure @juanpablob.

So, I have 4 environments configured, localhost, development, qa and production. Production is the only one that does not have a sub domain for it (all the others follow dev.domain.com, qa.domain.com , etc).

The problem is that when I on my QA environment, it's recognizing it as production. Same for DEV being recognized as production. If I comment out production, everything works. (So I guess it's a subdomain vs domain issue).

**Comment by juanpablob** [comment]
@marcussaad Oh! I never saw it come :D I'll write a fix and I'll let you know! Thanks again.

**Comment by juanpablob** [comment]
@marcussaad I just wrote a fix and released 1.0.3. Hope it helps. Cheers!

**Comment by marcussaad** [comment]
Thanks Juan! I'll take a look and let you know :D
